mysql-test/suite/binlog_gtid/t/binlog_gtid_log_status_mismatch.test

```diff
@@ -0,0 +1,37 @@
+--source include/have_debug.inc
+--source include/have_debug_sync.inc
+# This test case is binary log format agnostic
+--source include/have_binlog_format_row.inc
+
+connect (con1,localhost,root,,);
+--echo # Connection default
+connection default;
+RESET MASTER;
+
+CREATE TABLE t1(c1 INT);
+INSERT INTO t1 VALUES(1);
+INSERT INTO t1 VALUES(2);
+FLUSH BINARY LOGS;
+# The transaction will not commit until it get 'continue' signal.
+SET debug_sync = "process_commit_stage_queue_before_handle_gtid SIGNAL in_sync WAIT_FOR continue";
+--send INSERT INTO t1 VALUES(3)
+
+--echo # Connection 1
+connection con1;
+SET DEBUG_SYNC= 'now WAIT_FOR in_sync';
+SET DEBUG_SYNC= 'now SIGNAL continue';
+--send SELECT LOCAL->"$.gtid_executed" AS gtid_executed FROM performance_schema.log_status;
```

Comment:
Please add a wait_condition folowed by an assert here verifying that SELECT is waiting for the commit_lock.

We can use perfschema for validating this. We can use something like 
```
mysql> select PROCESSLIST_INFO from performance_schema.threads T, performance_schema.events_waits_current W where T.THREAD_ID=W.THREAD_ID AND W.EVENT_NAME LIKE '%LOCK_commit';
+-------------------------------------------------------------------------------------+
| PROCESSLIST_INFO                                                                    |
+-------------------------------------------------------------------------------------+
| SELECT LOCAL->"$.gtid_executed" AS gtid_executed FROM performance_schema.log_status |
+-------------------------------------------------------------------------------------+
1 row in set (0.01 sec)
```

```suggestion
--send SELECT LOCAL->"$.gtid_executed" AS gtid_executed FROM performance_schema.log_status;
< wait_condition.inc >
< assert.inc >
SET DEBUG_SYNC= 'now SIGNAL continue';
```

Reply:
done